### DISCUSSION TITLE
Add PT-BR Translation

### DISCUSSION THREAD

**Discussion by Phoenixx1202** [bucanero/apollo-ps3#69] [Ideas]
Is there the possibility of opening space to add the pt-br (Brazil) translation, similar to the pkgi project?

I would like to contribute.

Thanks

**Comment by bucanero** [comment]
> Is there the possibility of opening space to add the pt-br (Brazil) translation, similar to the pkgi project?
> 
> I would like to contribute.
> 
> Thanks

Apollo needs to be updated to support internationalization before adding any translations. When those changes are ready, I'll ask for translations and then you can submit the portuguese language file.

thanks for the message!
